resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +238,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class TransparencyEffects {
+    [DscProperty(Key)] [BinarySettingState] $TransparencyEffectsSetting = [BinarySettingState]::KeepCurrentValue
+
+    hidden [string] $TransparencySettingProperty = 'EnableTransparency'
+
+    [TransparencyEffects] Get() {
+        $currentState = [TransparencyEffects]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:TransparencyEffectsRegistryPath -Name $this.TransparencySettingProperty)) {
+            $TransparencySetting = [BinarySettingState]::Disabled                
+        } else {
+			$TransparencySetting = (Get-ItemProperty -Path $global:TransparencyEffectsRegistryPath -Name $this.TransparencySettingProperty).EnableTransparency
+			$currentState.TransparencyEffectsSetting = switch ($TransparencySetting) {
+				0 { [BinarySettingState]::Disabled }               
+				1 { [BinarySettingState]::Enabled }               
+				default { [BinarySettingState]::KeepCurrentValue }

```

Comment:
Avoid returning KeepCurrentValue as we want to show the use the exact state value.

Reply:
https://github.com/microsoft/winget-dsc/pull/49#discussion_r1718932728